src/Stratis.Bitcoin.Features.Wallet/Controllers/WalletController.cs

```diff
@@ -1482,6 +1480,189 @@ public IActionResult SplitCoins([FromBody] SplitCoinsRequest request)
             }
         }
 
+
+        /// <summary>Splits and distributes UTXOs across wallet addresses</summary>
+        [HttpPost]
+        [Route("distribute-utxos")]
+        public IActionResult DistributeUtxos([FromBody] DistributeUtxosRequest request)
```

Comment:
this is the DistributeUtxos I was talking about

Reply:
and this is the open PR that is trying to introduce that
https://github.com/stratisproject/StratisBitcoinFullNode/pull/3909

Reply:
I have conveyed @ArcSin2000X 's response to you. I am sure he will answer any follow-up questions you may still have.